tests/devices/qubit/test_dq_interpreter.py

```diff
@@ -167,13 +169,27 @@ def f(x):
     assert qml.math.allclose(output, expected)
 
 
+def test_parameter_broadcasting():
+    """Test that dq can execute a circuit with parameter broadcasting."""
+
+    @DefaultQubitInterpreter(num_wires=3)
+    def f(x):
+        qml.RX(x, 0)
+        return qml.expval(qml.Z(0))
+
+    x = jax.numpy.array([1.2, 2.3, 3.4])
+    output = f(x)
+    expected = jax.numpy.cos(x)
+    assert qml.math.allclose(output, expected)
```

Comment:
:heart_eyes:

Reply:
Does this work because `jax.vmap` is called under the hood?

Reply:
`jax.vmap` requires this kind of workflow to work, at least for now.  In the future we will need to figure out how to decompose parameter broadcasting into a for loop, in which case we will no longer need devices to support operators with batch dimensions, but for now this is an easy enough fix.

Reply:
DQ will natively support parameter broadcasting and jitting, but other devices (like lightning) won't.  We'll need to figure out where and when we need to invoke a `jax.pure_callback` boundary.